scraper/src/github-scraper/utils.ts

```diff
@@ -0,0 +1,142 @@
+import path from "path";
+import { octokit } from "./config.js";
+import { Action, ActivityData, PullRequestEvent } from "./types.js";
+import { promises as fs } from "fs";
+
+export const parseISODate = (isoDate: Date) => {
+  return new Date(isoDate);
+};
+
+const userBlacklist = new Set(["dependabot", "snyk-bot", "codecov-commenter"]);
+
+export const isBlacklisted = (login: string): boolean => {
+  return login.includes("[bot]") || userBlacklist.has(login);
+};
+
+export async function calculateTurnaroundTime(event: PullRequestEvent) {
+  const user: string = event.payload.pull_request.user.login;
+  const mergedAt: Date = parseISODate(event.payload.pull_request.merged_at);
+  const createdAt: Date = parseISODate(event.payload.pull_request.created_at);
+
+  const linkedIssues: [string, string][] = [];
+  const body = event.payload.pull_request.body || "";
+  const regex =
+    /(fix|fixes|fixed|close|closes|closed|resolve|resolves|resolved) ([\w\/.-]*)(#\d+)/gi;
+  let match: RegExpExecArray | null;
```

Comment:
instead of scanning the body, can we directly fetch the linked issues of the PR using graphql?